media-sound/teamspeak-server-bin/files/teamspeak.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=TeamSpeak3 Server
+After=network.target
+
+[Service]
+Type=simple
+Environment=LD_LIBRARY_PATH=/opt/teamspeak3-server
+WorkingDirectory=/opt/teamspeak3-server
+ExecStartPre=/bin/rm -f /dev/shm/7gbhujb54g8z9hu43jre8
+ExecStart=/opt/bin/ts3server inifile=/etc/teamspeak3-server/ts3server.ini
+ExecStop=/bin/kill -15 $MAINPID
```

Comment:
Why do you need to explicitly kill it like that? Doesn't systemd's default killing work for it?

Reply:
I've checked it now, works fine without it for me, so dropping it.